draft-ecahc-moderation.md

```diff
@@ -121,6 +121,14 @@ has, mostly for historic reasons, a team of moderators that are not
 list administrators and operate by a different set of processes
 {{?RFC9245}}.
 
+Note that the term "moderation" can refer both to *preemptive*
+moderation, where a moderator reviews messages before release
+to the mailing list, and *reactive* moderation, where the moderator
+intervenes after a message has been sent. The IETF mainly practices
```

Comment:
This is written for mailing lists, but the new moderation approach is supposed to cover all types of participation channels.